lib/picam/camera.ex

```diff
@@ -33,11 +40,19 @@ defmodule Picam.Camera do
 
   def handle_info({_, {:data, jpg}}, state) do
     Task.start(fn -> dispatch(state.requests, jpg) end)
-    {:noreply, %{state | requests: []}}
+    {:noreply, %{state | requests: [], offline: false}}
+  end
+
+  def handle_info(:reconnect_port, state) do
+    executable = Path.join(:code.priv_dir(:picam), "raspijpgs")
+    port = Port.open({:spawn_executable, executable}, [{:packet, 4}, :use_stdio, :binary, :exit_status])
```

Comment:
How do you feel about checking the return value here via pattern match and then calling `Process.send_after` again if it fails to run?  Otherwise I think this will only attempt to start the port once after the initial 10 seconds has passed.

Reply:
yeah sounds good